tests/testthat/test-drive_trash.R

```diff
@@ -0,0 +1,52 @@
+context("Trash files")
+
+## WARNING: this will empty your drive trash. If you do
+## not want that to happen, set EMPTY_TRASH = FALSE
+# EMPTY_TRASH = TRUE
+
+## NOTE if you do not currently have the files needed,
+## change run & clean below to TRUE to create files needed
+## (CAUTION, this will delete files that will interfere)
+
+
+nm_ <- nm_fun("-TEST-drive-trash")
+
+run <- FALSE
+clean <- FALSE
```

Comment:
I'm just starting to have two blocks like so:

```
## clean
if (FALSE) {
  # delete stuff
}

## setup
if (FALSE) {
  # create stuff
}
```

because in reality I just interactively execute those lines when needed. The nesting and objects `run` and `clean` seem like something we can abandon.